overlay/add-postgress-variables.yml

```diff
@@ -0,0 +1,37 @@
+# Our adaptation of the upstream add-postgress-variables.yml which allows for maintaining the domain names.
```

Comment:
Please use overlay (spruce) style instead of go-patch style.

Reply:
I kept this format to match upstream, since it is "upstream" driven and edited to match our requirements

Reply:
Let's skip this for now.